src/modules/MainMenu/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../../App.css'; 
+import { Navbar, Nav } from 'react-bootstrap';
+
+const MainMenu = (props) => (
+          <Navbar expand="lg" bg="dark" variant="dark" fixed="top">
+                <Navbar.Brand href="/" className="main-menu-1">Logo</Navbar.Brand>
+                <Nav className="mr-auto main-menu-2">
```

Comment:
Remove "mr-auto" class in order for the items to align left, or replace with "m-auto" to align center.